QualityControl/public/common/runModeHeader.js

```diff
@@ -0,0 +1,236 @@
+      if (runNumber === 'N/A') {
+        return;
+      }
+      await navigator.clipboard.writeText(runNumber);
+      model.notification.show('Run number copied to clipboard', 'success', 1500);
+    };
+
+    return h('.run-detail.flex-row.items-center.g1', {
+      id: 'run-number',
+    }, [
+      h('span.gray-darker', {
+        id: 'run-number-label',
+      }, 'Run:'),
+      h('span.text-no-select', {
+        id: 'run-number-value',
+        title: runNumber === 'N/A' ? 'No run number available' : 'Click to copy run number',
+        onclick: copyToClipboard,
+        style: runNumber !== 'N/A' ? 'cursor: pointer; user-select: none;' : 'cursor: default;',
+      }, runNumber),
+    ]);
+  }
+
+  /**
+   * Renders the status detail with dropdown
+   * @param {Model} model - root model of the application
+   * @param {FilterModel} filterModel - model that manages filter state
+   * @returns {vnode} - virtual node element
+   */
+  function renderRunStatus(model, filterModel) {
+    return h('.run-detail.flex-row.items-center.g1', [
+      h('span.gray-darker', 'Status:'),
+      h(
+        `span.${getStatusClass(model.runStatus)}`,
+        { id: 'run-status', style: 'font-weight:bold' },
+        model.runStatus || 'Unknown',
+      ),
+      renderStatusInfoDropdown(filterModel),
+    ]);
+  }
+
+  /**
+   * Renders the status info dropdown
+   * @param {Model} model - root model of the application
+   * @param {FilterModel} filterModel - model that manages filter state
+   * @returns {vnode} - virtual node element
+   */
+  function renderStatusInfoDropdown(filterModel) {
+    return h('.dropdown.mh1', {
+      class: filterModel._statusInfoOpen ? 'dropdown-open' : '',
+    }, [
+      renderInfoButton(filterModel),
+      filterModel._statusInfoOpen && renderDropdownMenu(),
+    ]);
+  }
+
+  /**
+   * Renders the info button
+   * @param {FilterModel} filterModel - model that manages filter state
+   * @returns {vnode} - virtual node element
+   */
+  function renderInfoButton(filterModel) {
+    return h('button.btn.btn-sm', {
+      title: 'Show status information',
+      onclick: (e) => {
+        e.stopPropagation();
+        filterModel._statusInfoOpen = !filterModel._statusInfoOpen;
+        filterModel.notify();
+      },
+      style: 'padding: 2px 6px; font-size: 0.75rem;',
+    }, info());
+  }
+
+  /**
+   * Renders the dropdown menu content
+   * @returns {vnode} - virtual node element
+   */
+  function renderDropdownMenu() {
+    return h('.dropdown-menu.show', [
+      h('.p2.text-left', [
+        h('span.gray-darker', 'Status meanings:'),
+        h('hr'),
+        renderStatusList(),
+      ]),
+    ]);
+  }
+
+  /**
+   * Renders the list of all possible statuses
+   * @returns {vnode} - virtual node element
+   */
+  function renderStatusList() {
+    return h('div', [
+      h('div', ...Object.values(RunStatus).map((status) =>
+        h('label', {
+          style: 'transition: transform 0.2s;',
```

Comment:
This transformation implies an action that can be taken by the user.
What is the action supposed to be in this `legend`?